src/components/flyout/OverlayHeader.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/variables';
+
+.bdl-OverlayHeader {
+    position: relative;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 64px;
+    padding: 13px 16px;
```

Comment:
Can we use the `$bdl-grid-unit` variables for these and go with 12px and 16px to standardize?